core/src/main/java/hivemall/utils/math/MathUtils.java

```diff
@@ -401,4 +401,13 @@ public static double logsumexp(@Nonnull final double[] arr, final double max) {
         return Math.log(logsumexp) + max;
     }
 
+    @Nonnull
+    public static float[] normalize(@Nonnull final float[] arr) {
```

Comment:
`sum` is l1norm? Then, remove this to `l1normalize`.

`l1norm` should be the sum of the **absolute** values of the columns.